src/App.tsx

```diff
@@ -1,9 +1,53 @@
 import './App.scss';
+import { useState, useEffect } from 'react';
+import {
+  Routes,
+  Route,
+  NavLink,
+  Navigate,
+} from 'react-router-dom';
+import { getPeople } from './api';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { Person } from './types/Person';
+import { NotFoundPage } from './components/NotFoundPage';
 
-export default App;
+export const App = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const loading = async () => {
+      const data = await getPeople();
+
+      setPeople(data);
+    };
+
+    loading();
+  }, []);
```

Comment:
Fetch people from [API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened, move this logic to PeoplePage